<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second</class>
 <widget class="QDialog" name="second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>گلخانه</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/background.png);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>101</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelName1">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/new/prefix1/assets/lock0.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton1">
      <property name="text">
       <string>بازگشایی</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton1">
      <property name="text">
       <string>ورود به فروشگاه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton1">
      <property name="text">
       <string>چیدن گل</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>101</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelName2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/new/prefix1/assets/lock0.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton2">
      <property name="text">
       <string>بازگشایی</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton2">
      <property name="text">
       <string>ورود به فروشگاه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton2">
      <property name="text">
       <string>چیدن گل</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>101</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="labelName3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/new/prefix1/assets/lock0.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton3">
      <property name="text">
       <string>بازگشایی</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton3">
      <property name="text">
       <string>ورود به فروشگاه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton3">
      <property name="text">
       <string>چیدن گل</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>101</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="labelName4">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/new/prefix1/assets/lock0.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton4">
      <property name="text">
       <string>بازگشایی</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton4">
      <property name="text">
       <string>ورود به فروشگاه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton4">
      <property name="text">
       <string>چیدن گل</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>101</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="labelName5">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resource.qrc">:/new/prefix1/assets/lock0.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton5">
      <property name="text">
       <string>بازگشایی</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton5">
      <property name="text">
       <string>ورود به فروشگاه</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saleButton5">
      <property name="text">
       <string>چیدن گل</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="openButton1_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
